### PR TITLE
Adding MigrateWebSdkRule to the DefaultMigrationRuleSet

### PR DESCRIPTION
**Pull request by livarcocc**
Adding MigrateWebSdkRule to the DefaultMigrationRuleSet and adding a E2E test to cover it.

Fixes https://github.com/dotnet/cli/issues/4962.

@piotrpMSFT @jgoshi @krwq 

@MattGertz for approval.

### CODE REVIEW

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 1]
```diff
@@ -126,9 +122,27 @@ public void It_migrates_old_dotnet_new_web_without_tools_with_outputs_containing
...
+            var projectDirectory = testInstance.Path;
+
+            var globalDirectory = Path.Combine(projectDirectory, "..");  
+            var projectJsonFile = Path.Combine(projectDirectory, "project.json");  
+              
+            MigrateProject(new [] { projectDirectory });
+
+            var csProj = Path.Combine(projectDirectory, $"{new DirectoryInfo(projectDirectory).Name}.csproj");
+
+            File.ReadAllText(csProj).Should().Contain(@"Sdk=""Microsoft.NET.Sdk.Web""");
```
Comment (piotrpMSFT) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Can we make sure this also builds happily?

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 2]
```diff
@@ -108,10 +107,7 @@ public void It_migrates_old_dotnet_new_web_without_tools_with_outputs_containing
             var globalDirectory = Path.Combine(projectDirectory, "..");  
             var projectJsonFile = Path.Combine(projectDirectory, "project.json");  
               
-            WriteGlobalJson(globalDirectory);  
-            var projectJson = JObject.Parse(File.ReadAllText(projectJsonFile));  
-            projectJson.Remove("tools");  
-            File.WriteAllText(projectJsonFile, projectJson.ToString());  
+            WriteGlobalJson(globalDirectory);
```
Comment (piotrpMSFT) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Did we now ensure the template doesn't have a tools node?

Comment (livarcocc) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: No. And I don't think we should. What we talked about and I will take care of in another PR, is make sure that none of the templates will have a cyclic dependency back in the CLI.

Comment (piotrpMSFT) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: ok

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs (hunk 1 above)
Comment (piotrpMSFT) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Thanks for enabling it!

**Comment by piotrpMSFT** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Windows_NT x64 Release Build please

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by MattGertz** [comment]
Yes, sounds like we need to take this.

**Comment by livarcocc** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by mlorbetske** [comment]
:shipit:

**Pull request merged**